torch/nn/utils/parametrize.py

```diff
@@ -282,6 +284,21 @@ def _inject_new_class(module: Module) -> None:
     """
     cls = module.__class__
 
+    def default_deepcopy(self, memo):
+        # Just emulate a standard deepcopy procedure when __deepcopy__ doesn't exist in the current class.
+        if id(self) in memo:
+            return memo[id(self)]
```

Comment:
Nit. We may want to go with the more efficient:
```python
obj = memo.get(id(self))
if obj is not None:
  return obj
```

Reply:
Agree, changed to proposed code to remove indentation :heavy_check_mark: